testsdata/prefix/a

```diff
@@ -0,0 +1 @@
+a/a1.in
```

Comment:
This PR adds some files under the testsdata directory. I think @RobiNino renamed the testsdata directory in a different PR, which hasn't been merged yet, so I suggest you sync with him to avoid recreating the directory he renamed.